compiler/rustc_mir_transform/src/lib.rs

```diff
@@ -242,6 +243,8 @@ fn mir_const<'tcx>(
             &Lint(function_item_references::FunctionItemReferences),
             // What we need to do constant evaluation.
             &simplify::SimplifyCfg::new("initial"),
+            // FIXME IN REVIEW: This is probably not the correct place for this.
+            &check_maybe_uninit::CheckMaybeUninit,
```

Comment:
I don't think there's anything wrong with putting it here. It doesn't really matter. I guess we want this to block optimizations, so putting it before that is the only important thing.